libs/common/src/constants/amendments.ts

```diff
@@ -50,6 +50,8 @@ export const AMENDMENT_QUERIES = {
   LATEST: 'latest',
 } as const;
 
+export const DEALIDS_CHECKERS_APPROVAL_AMENDMENTS = 'dealIdsCheckersApprovalAmendments';
```

Comment:
```suggestion
export const CHECKERS_AMENDMENTS_DEAL_ID = 'checkersAmendmentDealId';
```

what are yout thoughts on above please?